client/app/components/project-milestone.js

```diff
@@ -32,30 +32,4 @@ export default class ProjectMilestoneComponent extends Component {
 
     return 'future';
   }
-
-  @computed('tense', 'milestone.{displayDate,displayDate2}')
-  get timeRelativeToNow() {
```

Comment:
This was not used anywhere, so deleting.

Reply:
ty